pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/EventConfigurationDto.java

```diff
@@ -105,6 +111,14 @@ public long getAnnouncementMsgTemplateId() {
         return announcementMsgTemplateId;
     }
 
+    public boolean getAnnouncementIsPermanent() {
```

Comment:
This might be something for process meeting--I tend to prefer `isPermanent()` or `shouldCreateAnnouncementForProxies()` instead of sticking `get` in front of method names.

Reply:
I do too, but this DTO is a hodgepodge or attributes, and writing `eventDto.isPermanent()` doesn't really make sense, so I used the `get` naming here to indicate we're just getting some flag out of the DTO.